pkg/imgutil/qemuimgutil/qemuimgutil.go

```diff
@@ -16,6 +16,28 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+const QEMUIMGFORMAT = "qcow2"
```

Comment:
Please see https://go.dev/wiki/CodeReviewComments#mixed-caps

```suggestion
const DefaultQemuImgFormat = "qcow2"
```